sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -14,13 +15,21 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
     },
 
     markAllAsRead: function() {
+        this.stopListening(this.collection, 'change:unread');
+        this.model.save({unread: false});
```

Comment:
We need to save since we need the database to sync against the UserFeed/UserCategory table now. Maybe the better way to do it is disable the button or make it only listenOnce to the event (Backbone has that built in)